dev-app/src/util/merchantStorage.ts

```diff
@@ -80,6 +81,22 @@ export const getSelectedAccount = async (): Promise<string | null> =>
 export const setSelectedAccount = async (accountKey: string) =>
   await AsyncStorage.setItem(SELECTED_ACCOUNT_KEY, accountKey);
 
+export const getEnableAutoReconnect = async (): Promise<boolean | null> => {
+  try {
+    const value = await AsyncStorage.getItem(AUTOMATIC_RECONNECTION_KEY);
+    return value !== null ? JSON.parse(value) : null;
+  } catch (error) {
+    console.log(error);
+    return null;
+  }
+};
+
+export const setEnableAutoReconnect = async (autoReconnection: boolean) =>
+  await AsyncStorage.setItem(
+    AUTOMATIC_RECONNECTION_KEY,
+    JSON.stringify(autoReconnection)
+  );
```

Comment:
asyncStorage seems like a bit of an overkill to persist this. maybe we can leverage AppContext instead https://github.com/stripe/stripe-terminal-react-native/blob/4bfa8622/dev-app/src/AppContext.ts#L8-L14